product_docs/docs/jdbc_connector/42.5.4.2/installing/linux_ppc64le/index.mdx

```diff
@@ -27,5 +27,3 @@ Operating system-specific install instructions are described in the correspondin
 ### SUSE Linux Enterprise (SLES)
 
 - [SLES 15](jdbc_sles_15)
-
-- [SLES 12](jdbc_sles_12)
```

Comment:
Need to remove SLES 12 from navigation list as well.

Reply:
done